src/App.css

```diff
@@ -0,0 +1,49 @@
+*{
+  margin: 0 auto;
```

Comment:
why `0 auto`?

Reply:
Because I need all the data on the website to be in the middle horizontally.

Reply:
use padding thn